pullupclub/models.py

```diff
@@ -18,5 +18,17 @@ class PullUpSession(models.Model):
     def __str__(self):
         return ("No. " + str(self.pk))
 
-#class Awards(models.Model):
-#    type
\ No newline at end of file
+class Award(models.Model):
+    TYPE = (
+        ('BD', 'By Distance'),
+        ('BC', 'By Count'),
+    )
+    name = models.CharField(max_length=60)
+    type = models.CharField(max_length=2, choices=TYPE)
+    arm_length_inches = models.PositiveSmallIntegerField(default=0)
```

Comment:
arm_length_inches should be on the student model

Reply:
Hmmm,
I see. 
This is kind of different from what I envisioned in the beginning.
Not that I don't like it, but let's talk about it.

Because I know there are different types of pullups:
Different grips, half-sies, negatives.
is arm_length supposed to cover that?

And idk If I will measure the length of arms for all students lol.
Also I feel like longer arm, more proportional muscles you have so.... biased against short limber ppl?

Reply:
Think of it this way. Does an award have arms with a length? No, a student does. If a student does 5 pull-ups then the distance they traveled is their own arm length * pull up count. This has nothing to do with the award.  You can do a default, or average for all students. I see what you saying about the grips, but let’s ignore that for now.